lib/stdlib/Duden/Listen.ddp

```diff
@@ -486,6 +516,34 @@ Die öffentliche Funktion Produkt_Kommazahlen_Liste mit dem Parameter liste vom
 Und kann so benutzt werden:
 	"das Produkt aller Kommazahlen in <liste>"
 
+[
+	Gibt die kleinste Kommazahl in der gegebenen Kommazahlen Liste zurück.
+	l := {e1, e2, e3, ..., en}
+	f(l) = min(e1, e2, e3, ..., en)
+	f({}) = 0,0
+]
+Die öffentliche Funktion Minimum_Kommazahlen_Liste mit dem Parameter liste vom Typ Kommazahlen Liste, gibt eine Kommazahl zurück, macht:
+	Wenn liste leer ist, gib 0,0 zurück.
+	Die Kommazahl min ist der maximale Wert einer Kommazahl.
+	Für jede Kommazahl z in liste, wenn z kleiner als min ist, speichere z in min.
+	Gib min zurück.
+Und kann so benutzt werden:
+	"die kleinste Kommazahl in <liste>"
+
+[
+	Gibt die größte Kommazahl in der gegebenen Kommazahlen Liste zurück.
+	l := {e1, e2, e3, ..., en}
+	f(l) = max(e1, e2, e3, ..., en)
+	f({}) = 0,0
+]
+Die öffentliche Funktion Maximum_Kommazahl_Liste mit dem Parameter liste vom Typ Kommazahlen Liste, gibt eine Kommazahl zurück, macht:
+	Wenn liste leer ist, gib 0,0 zurück.
+	Die Kommazahl max ist der minimale Wert einer Kommazahl.
+	Für jede Kommazahl z in liste, wenn z größer als max ist, speichere z in max.
+	Gib max zurück.
+Und kann so benutzt werden:
+	"die größte Kommazahl in <liste>"
+
```

Comment:
Diese Funktionen existieren schon in Duden/Statistik. https://doku.ddp.im/Programmierung/Standardbibliothek/Statistik/